pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -434,14 +448,24 @@ func volumeRequiresFSResize(pvc *v1.PersistentVolumeClaim, pv *v1.PersistentVolu
 }
 
 // podPreviouslyProcessed returns true if the volumes for this pod have already
-// been processed by the populator
+// been processed/reprocessed by the populator. Otherwise, the volumes for this pod need to
+// be reprocessed.
 func (dswp *desiredStateOfWorldPopulator) podPreviouslyProcessed(
 	podName volumetypes.UniquePodName) bool {
 	dswp.pods.RLock()
 	defer dswp.pods.RUnlock()
 
-	_, exists := dswp.pods.processedPods[podName]
-	return exists
+	return dswp.pods.processedPods[podName]
+}
+
+// podHasBeenSeenOnce returns true if the pod has been seen by the popoulator
+// at least once.
+func (dswp *desiredStateOfWorldPopulator) podHasBeenSeenOnce(
+	podName volumetypes.UniquePodName) bool {
+	dswp.pods.RLock()
+	_, exist := dswp.pods.processedPods[podName]
+	dswp.pods.RUnlock()
```

Comment:
```suggestion
	dswp.pods.RLock()
	defer dswp.pods.RUnlock()
	_, exist := dswp.pods.processedPods[podName]
```

Reply:
Since there is no chance of panic in accessing the map, I choose the (slightly) more efficient way of releasing the lock.
I prefer to keep this form.

Reply:
I would prefer to lean towards safety rather then performance in this tradeoff. But will defer to you as PR author.